litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/index.tsx

```diff
@@ -285,9 +289,9 @@ const TuneWorkflow: React.FC = () => {
           {/* Workflow Table */}
           <Width width="70%">
             {experiment.length > 0 ? (
-              <WorkflowTable isCustom crd={manifest} />
+              <WorkflowTable isCustom />
             ) : (
-              <WorkflowTable isCustom={false} crd={workflow.crd} />
+              <WorkflowTable isCustom={false} />
```

Comment:
doesn't make isCustom a required field...as undefined and false will both act as falsey values

Reply:
Will make this change 👍🏻